### PR TITLE
Misc copy edits

### PR DESCRIPTION
**Pull request by pabloest**
for clarity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+Contributions to the docs are welcome! To start contributing, first please make sure you read and agree with the Semgrep’s [Contributor Covenant Code of Conduct](https://github.com/returntocorp/semgrep/blob/develop/CODE_OF_CONDUCT.md).
+
+### Developing docs locally
 1. Install mkdocs: `pip install mkdocs`
-2. Install required plugins: `pip install mkdocs-redirects`
-3. Clone the repo
-4. Run the docs locally with `mkdocs serve` and then go to: http://127.0.0.1:8000/
+3. Install required plugins: `pip install mkdocs-redirects`
+4. Clone the repo
+5. Run the docs locally with `mkdocs serve` and then go to: http://127.0.0.1:8000/
```
Comment (underyx) [inline README.md:20]: Numbering seems off.

Comment (pabloest) [reply README.md:20]: Indeed, thanks for catching that. Fixing now.

**Comment by pabloest** [comment]
Could you approve, @underyx?

**Comment by dlukeomalley** [comment]
@pabloest Still relevant, and if so, up for resolving conflicts?

**Comment by pabloest** [comment]
Probably still relevant and yes, I can look into the conflicts.

**Comment by pabloest** [comment]
@dlukeomalley should be good to go now

**Review by dlukeomalley — APPROVED** [review]

**Pull request merged**